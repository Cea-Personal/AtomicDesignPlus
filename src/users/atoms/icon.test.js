import React from 'react';
import Icon from './icon'
import renderer from 'react-test-renderer';

test('should render a word', () => {
    const tree = renderer.create(<Icon/>).toJSON()
    expect(tree).toMatchSnapshot()
})