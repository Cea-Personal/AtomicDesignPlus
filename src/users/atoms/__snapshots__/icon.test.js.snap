// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows the word Lifted 1`] = `
<DocumentFragment>
  <p
    class="sc-bdVaJa jvkMKG"
  >
    Lifted
  </p>
</DocumentFragment>
`;
